code/modules/hydroponics/hydro_tray.dm

```diff
@@ -771,7 +772,28 @@
 		remove_dead(user)
 
 	else
-		view_contents(user)
+		examine(user) //using examine() to display the reagents inside the tray as well
+
+/obj/machinery/portable_atmospherics/hydroponics/examine(mob/user)
+	..()
+	view_contents(user)
+
+/obj/machinery/portable_atmospherics/hydroponics/proc/view_contents(mob/user)
+	if(src.seed && !src.dead)
+		user << "[src] has <span class='notice'>[src.seed.display_name]</span> planted."
```

Comment:
Examine text normally uses the 'info' span class instead of 'notice' which is meant to be used for information that has just occurred (like completing an action)